source/jormungandr/jormungandr/authentication.py

```diff
@@ -161,7 +161,7 @@ def get_user(token, abort_if_no_token=True):
             #a token is mandatory for non public jormungandr
             if not current_app.config.get('PUBLIC', False):
                 if abort_if_no_token:
-                    flask_restful.abort(401, message='no token')
+                    flask_restful.abort(401, message='no token. You can get one at http://www.navitia.io')
```

Comment:
I think this should be a parameter, because I don't think self hosted versions will like this message :stuck_out_tongue_winking_eye: